engine/apps/email/inbound.py

```diff
@@ -26,12 +27,14 @@ class AmazonSESValidatedInboundWebhookView(amazon_ses.AmazonSESInboundWebhookVie
 
     def validate_request(self, request):
         """Add SNS message validation to Amazon SES inbound webhook view, which is not implemented in Anymail."""
-
-        super().validate_request(request)
```

Comment:
call to `super().validate_request` is unnecesery here because django-anymail collects these `validate_request` methods from the whole superclass chain and runs them all in `run_validators`: https://github.com/anymail/django-anymail/blob/35383c7140289e82b39ada5980077898aa07d18d/anymail/webhooks/base.py#L28